graph-traversal/test_graphtrav.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+GRAPH = {'A': ['B', 'C'], 'B': ['D', 'C'], 'C': [], 'D': ['E'], 'E': []}
+
+VISITED_B = ['A', 'B', 'C', 'D', 'E']
+
+VISITED_D = ['A', 'C', 'B', 'D', 'E']
+
```

Comment:
Why would your depth-first search go to "C" before "B"? Doesn't it always start from the left?